src/Voron/Data/Tables/Table.cs

```diff
@@ -1585,7 +1605,18 @@ public IEnumerable<TableValueHolder> SeekForwardFrom(TableSchema.FixedSizeSchema
                 var result = new TableValueHolder();
                 do
                 {
-                    GetTableValueReader(it, out result.Reader);
+                    try
+                    {
+                        GetTableValueReader(it, out result.Reader);
+                    }
+                    catch (InvalidOperationException e)
```

Comment:
cc @redknightlois 

This might cause some perf degradation due to the trycatch block here?

Can we use:
```
if (_onCorruptedDocumentErrorHandler == null)
    oldCode
else
    trycatch
````
?

Reply:
I've implemented everything for now through 

```C#
if (_onCorruptedDocumentErrorHandler == null)
    oldCode
else
    trycatch
```

If @redknightlois has no objections.